README.md

```diff
@@ -13,6 +13,9 @@ For installing the Identity Service you can choose Kubernetes or the distributio
 #### Overview
 This guide helps you get started with the Identity Service. It covers simple standalone startup and use of the default database. Advanced deployment options are not covered. For a deeper description of keycloak features or configuration options, consult the official [Keycloak readme](https://www.keycloak.org/docs/) .
 
+#### Prerequisites
+  1. Java 8 JDK
```

Comment:
I may be misremembering, but I thought this was working on Java 11 as well, perhaps with some additional flag?

Reply:
@rgauss Yes, it should run on either 8 or 11